src/Agent.Worker/Release/ReleaseCommandExtension.cs

```diff
@@ -12,76 +12,68 @@
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
 {
-    public sealed class ReleaseCommandExtension : AgentService, IWorkerCommandExtension
+    public sealed class ReleaseCommandExtension : BaseWorkerCommandExtension
     {
-        public Type ExtensionType => typeof(IWorkerCommandExtension);
-
-        public string CommandArea => "release";
-
-        public HostTypes SupportedHostTypes => HostTypes.Release | HostTypes.Deployment;
-
-        public void ProcessCommand(IExecutionContext context, Command command)
+        public ReleaseCommandExtension()
         {
-            if (string.Equals(command.Event, WellKnownReleaseCommand.UpdateReleaseName, StringComparison.OrdinalIgnoreCase))
-            {
-                ProcessReleaseUpdateReleaseNameCommand(context, command.Data);
-            }
-            else
-            {
-                throw new Exception(StringUtil.Loc("RMCommandNotFound", command.Event));
-            }
+            CommandArea = "release";
```

Comment:
I assume this is for telemetry?

Reply:
I guess, I was just trying to make the Exception string match what was there before